backends/ebpf/ebpftargets.py

```diff
@@ -23,24 +23,25 @@
 #   5. Evaluates the output with the expected result from the .stf file
 
 from __future__ import print_function
+import os
+import sys
+import subprocess
 from subprocess import Popen
-from threading import Thread
+from threading import Timer
 from glob import glob
-try:
-    from scapy.layers.all import *
-    from scapy.utils import *
-except ImportError:
-    pass
+from scapy.layers.all import RawPcapWriter
+from scapy.utils import rdpcap
+from pyroute2 import IPRoute
 
 SUCCESS = 0
 FAILURE = 1
 TIMEOUT = 10 * 60
 
 
-def nextWord(text, sep=None):
-    ''' Split a text at the indicated separator.
+def next_word(text, sep=None):
```

Comment:
Just renaming functions that are otherwise identical will make it very hard to remove this duplicated code later if we decide to refactor it.